taxcalc/utils.py

```diff
@@ -263,33 +269,36 @@ def results(c):
 
 
 def weighted_avg_allcols(df, cols):
-    diff = DataFrame(df.groupby('bins').apply(weighted_mean, "c00100"),
-                     columns=['c00100'])
+    diff = DataFrame(df.groupby('bins')['s006'].sum(), columns=['s006'])
```

Comment:
You are right that we always want to see the sum of s006 instead of the weighted average. (In this average table, four weights related variables are summed up and all others are taken weighted average.) So here I use the sum() function for the s006 column in the groupby object and I thought this step would return the sum instead of the weighted average. Do I understand this sum() function right?
The reason why I moved s006 to the first column is that 1) it's in line with the diagnostic table format 2) it's easier to compare with SOI tables, which always list the number of returns for each bin before the amount of tax/credit.